packages/deprecation-crawler/sandbox/deprecation-comments/input.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Examples in this file should multiple commenting styles.
+ */
+
+/** @deprecated comment-style single line deprecation */
+function foo() {
+  return 'foo';
+}
+
+/**
+ * @nocollapse
+ * @deprecated comment-style deprecation with leading text
+ */
+function foo2() {
+  return 'foo2';
+}
+
+/**
+ * This is foo3
+ * @method foo3
+ * @deprecated comment-style deprecation with leading and trailing text
+ * @return {void}
+ */
+function foo3() {
+  return 'foo3';
+}
+
+/** @deprecated comment-style single line deprecation with no space at the end*/
```

Comment:
Do we also need to test // commented lines?

Reply:
Good question, let's try it out.

Reply:
It's covered 🎉